core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.spec.ts

```diff
@@ -914,10 +923,15 @@ describe('Settings Tab Component', () => {
         name: 'test',
       };
 
+      ctrl.refreshSettingsTab();
+
+      tick(1000);
```

Comment:
why is this 1000 if the setTimeout is only 500?

Reply:
Fixed.